src/java.base/share/classes/jdk/internal/misc/InnocuousThread.java

```diff
@@ -50,52 +50,82 @@ private static String newName() {
     }
 
     /**
-     * Returns a new InnocuousThread with an auto-generated thread name
-     * and its context class loader is set to the system class loader.
+     * Returns a new InnocuousThread with an auto-generated thread name,
+     * inheriting the current thread priority, and its context class loader
+     * is set to the system class loader.
      */
     public static Thread newThread(Runnable target) {
         return newThread(newName(), target);
     }
 
     /**
      * Returns a new InnocuousThread with its context class loader
-     * set to the system class loader.
+     * set to the system class loader, inheriting the current thread priority
      */
```

Comment:
Needs a period at the end of the sentence.  Which is getting too long for the method summary.
(Even though javadoc is not usually generated for this implementation class).
Split the qualification about thread priority into a separate sentence.